htdocs/langs/fr_FR/admin.lang

```diff
@@ -822,6 +822,7 @@ DictionaryUnits=Unités
 DictionaryProspectStatus=Statuts de prospection
 DictionaryHolidayTypes=Type de congès
 DictionaryOpportunityStatus=Statut d'opportunités pour les affaires/projets
+DictionaryFieldCases=Mise en forme forcée de certains champs (majuscules, camelCase, RegEx...)
```

Comment:
French translations should be contributed at [Transifex](https://www.transifex.com/dolibarr-association/dolibarr/dashboard/) not appear in commits.
On the other hand en_US are required here.